data/members.json

```diff
@@ -34,4 +34,12 @@
   { "name": "Mike Elsmore", "feed": null, "url": "https://elsmore.me/" },
   { "name": "Martin Hicks", "feed": null, "url": "https://martinhicks.net/" },
   { "name": "William Hermozo", "feed": "https://williamhzo.me/rss.xml", "url": "https://williamhzo.me/" }
+  [
```

Comment:
Please remove the array brackets around your entry!

Reply:
Sorry about that! Should be fixed now.